src/screens/fareResults.tsx

```diff
@@ -57,7 +59,7 @@ const FareResultsScreen: React.FC<FareResultsProps> = ({ route }) => {
 
   return (
     <View style={styles.container}>
-      <Text style={styles.text}>Select Stations</Text>
+      <Text style={styles.body}>Journeys Available</Text>
       {fareResponseData ? (
```

Comment:
Not part of this change, but a good thing to think about here for a future modification is what happens if the fare response comes back as an empty list - this is a valid edge case that can happen. What would your UI want to do in this case to avoid confusing the user?